benchtop/src/nomt.rs

```diff
@@ -169,5 +169,6 @@ impl<'a> Transaction for Tx<'a> {
         }
 
         self.session.warm_up(key_path);
+        self.session.preserve_prior_value(key_path);
```

Comment:
just a note (as an issue that existed already)

if we write the same slot twice, these calls are redundant. but our current workloads typically won't.